Cargo.toml

```diff
@@ -20,11 +20,11 @@ faster-hex = "0.3"
 
 [dev-dependencies]
 byteorder = "1.3.1"
-ckb-core = { git = "https://github.com/nervosnetwork/ckb.git", rev = "6f5545d68eab0521520ceb98d8be623dc6d9c8a7" }
-ckb-script = { git = "https://github.com/nervosnetwork/ckb.git", rev = "6f5545d68eab0521520ceb98d8be623dc6d9c8a7" }
-ckb-crypto = { git = "https://github.com/nervosnetwork/ckb.git", rev = "6f5545d68eab0521520ceb98d8be623dc6d9c8a7" }
-ckb-dao-utils = { git = "https://github.com/nervosnetwork/ckb.git", rev = "6f5545d68eab0521520ceb98d8be623dc6d9c8a7" }
-ckb-hash = { git = "https://github.com/nervosnetwork/ckb.git", rev = "6f5545d68eab0521520ceb98d8be623dc6d9c8a7" }
+ckb-core = { git = "https://github.com/jjyr/ckb.git", rev = "6dd8df44566c7f3035cafc3774cacbcfbc9add7d" }
```

Comment:
Can we push the branch to the main repo and keep using the upstream version instead of a fork? Just want to make sure we didn't forget to change this before merging.

Reply:
Sure, but I need to publish this crate first, then to update the ckb PR with the new version contract, then update this commit id after the ckb PR get merged.

Reply:
Yes I've actually been there in #21. You can first tag the commit in this PR with rc tags, then you can use the rc version in CKB PR.